docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,153 +2,150 @@
-> If you do you can refer to this guide to troubleshoot common errors, or reach out to us on our
+> If you do, refer to this guide to troubleshoot common errors, or reach out to us on our
 > [Telegram channel](https://t.me/antelopedevs) for help.
 
-You can see a list of EOS nodes on your system:
+The following command shows you a list of EOS nodes on your system:
 
 ```shell
 dune --list
 ```
 
-You can also check if your active node's RPC API is live:
+The following command checks if your active node's RPC API is live:
 
 ```shell
 dune -- cleos get info
 ```
 
-To shut down your node:
+The following command shuts down your node:
 
 ```shell
 dune --stop <NODE_NAME>
 ```
 
-To remove a node entirely:
+The following command removes a node:
 
 ```shell
 dune --remove <NODE_NAME>
 ```
 
-
 ### Bootstrapping your environment
 
-There are a few system contracts that your development environment might need to rely on such as:
+There are a few system contracts that your development environment may need to rely on, such as:
 - `eosio.token` for **EOS** token transfers
 - `eosio.msig` for multisig transactions
 - `eosio.system` for system level actions such as resource management
 
-Bootstrapping your local node is easy, once you have an active node running, you can bootstrap it with:
+Bootstrapping your local node is easy. Once you have an active node running, bootstrap it with:
 
 ```shell
 dune --bootstrap-system-full
 ```
 
-
 ## Account management
 
-You deploy contracts on top of accounts, and also use them to interact with your smart contracts. 
+You deploy contracts on top of accounts. Use accounts to interact with your smart contracts. 
 
-To create a new account:
+The following command creates a new account:
 
 ```shell
 dune --create-account <ACCOUNT_NAME>
 ```
 
-To get account info:
+The following command gets account info:
 
 ```shell
 dune -- cleos get account <ACCOUNT_NAME>
 ```
+## Smart contract development
 
-## Smart Contract Development
+Create a sample project to learn how to use DUNE to compile, deploy, and interact with smart contracts.
 
-Let's create a sample project so that we can learn how to compile, deploy, and interact with smart contracts using DUNE.
+### Create sample project
 
-Navigate to a directory you want to create a project in, and then run the following command:
+Navigate to a directory in which you want to create a project, then run the following command:
 
 ```shell
 dune --create-cmake-app hello .
 ```
-
-This will create a `hello` directory with a cmake style EOS smart contract project.
+This command creates a `hello` directory with a cmake style EOS smart contract project.
 
 Replace the contents of `src/hello.cpp` with the following code:
 
 ```cpp
-#include <eosio/eosio.hpp>
+#include <eosio/eosio.hpp> 
 using namespace eosio;
 
-CONTRACT hello : public contract {
-   public:
-      using contract::contract;
+CONTRACT hello : public contract {ompilinc
```

Comment:
Not sure what happened here, but please don't touch code blocks. 

Indentation changed, and `ompilinc` was added here

Reply:
I primarily work with text. What needs to be corrected?

Reply:
All of the indentation of the code block was messed up and there is an added `ompilinc` text there which breaks the code.

Reply:
Done